config/kubernetes/sig-openstack/teams.yaml

```diff
@@ -2,36 +2,42 @@ teams:
   provider-openstack-api-reviews:
     description: ""
     maintainers:
-    - idvoretskyi
+    - lingxiankong
```

Comment:
@lingxiankong -- You'll need to put your name in the `members` list. 
Only members of the [GitHub Administration Team](https://github.com/kubernetes/community/tree/master/github-management#github-administration-team) should be listed in the `maintainers` section (or else the tests will fail).

Reply:
Thanks, will change, actually I'm not quite sure what these groups in this file are meant for.